recipes-core/base-files/base-files/locale.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```

Comment:
please rename the file to `profile.d_locale.sh` so it is clear where it is being installed.